### ISSUE TITLE
Try out LayerNorm instead of BatchNorm

### ISSUE THREAD

**Issue by gcroci2** [DeepRank/3D-Vac#145]
We could try out [LayerNorm](https://pytorch.org/docs/stable/generated/torch.nn.LayerNorm.html) instead of BatchNorm1d (issue #131), which may work better and require less training time compared to BatchNorm1d.
